packages/web-components/@sfx/sayt/CHANGELOG.md

```diff
@@ -8,3 +8,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - SFX-191: Added the `sayt` component.
   - This component renders `sayt` related components such as `sfx-autocomplete` and `sfx-products`.
+- SFX-253: Added `sfx-products` to dispay autocomplete products.
```

Comment:
I thought I had flagged this before, but since this is still unreleased, we can add this as a sub-point, or even not at all because it's covered by line 10.

Reply:
Removing since Daniel also suggested it be removed.